db/multi_cf_iterator_impl.h

```diff
@@ -85,7 +85,7 @@ class MultiCfIteratorImpl {
   void Next() {
     assert(Valid());
     auto& min_heap = GetHeap<MultiCfMinHeap>([this]() {
-      Slice target = key();
+      std::string target(key().data(), key().size());
```

Comment:
Copying keys can be a significant source of performance bugs, but I suppose this is only done when changing iterator direction, which is rare. And not sure we have an alternative.

Reply:
Yeah, I hope this to be a rare operation.